sdks/how-to/how-to-filter-data-with-advanced-search-criteria.md

```diff
@@ -336,19 +467,29 @@ const ageGenderImplicitPatients = await api.patientApi.filterPatients(ageGenderI
 ```
 </details>
 
-In this case, the method will return all the patients that the hcp with id `healthcarePartyId` can access, whose age is `42`, and whose gender is `female`.
-You can also explicitly intersect simple filters using the `intersection()` method:
+In this case, the method will return all the patients that the hcp with id `healthcarePartyId` can access, born between the
+11th of December 1939 and the 3rd of February 1952, and whose gender is `female`. The result will be sorted by date of 
+birth in ascending order.
```

Comment:
```suggestion
birth in ascending order. If you don't specify the sorting key the data may still be ordered according to some field, but we do not guarantee that this will be a consistent behaviour. 

::: info

In complex situations sorted filters may be less efficient than unsorted filters. You should not request sorting unless you really need sorted data. 

:::
```